src/applications/virtual-agent/components/webchat/WebChat.jsx

```diff
@@ -17,6 +18,18 @@ import {
 
 const renderMarkdown = text => MarkdownRenderer.render(text);
 
+export function cardActionMiddleware(next, card) {
+  // Track decision letter downloads
+  if (card.cardAction.value.includes('/v0/claim_letters/')) {
```

Comment:
Can the condition be tied to something more robust like an action identifier/name as opposed to the url?

I just feel like it would be more scalable that way.

Reply:
When we were looking at the input, when the card was an adaptive card, we didn't see anything like that. We tried adding an ID to the adaptive card and it didn't come through here. :(

Reply:
We may have more things we can do if we move to latest Adaptive cards

Reply:
With the technique we chose, this is the only way at the moment.
We can reengineer using on-clicks which would be more specific

Reply:
We can wait for the adaptive cards update to explore better ways. Thanks!